src/main/java/com/cgdp/library/validation/Validator.java

```diff
@@ -0,0 +1,42 @@
+package com.cgdp.library.validation;
+
+import static com.cgdp.library.validation.IsbnValidator.isNullOrBlank;
+import static com.cgdp.library.validation.IsbnValidator.isValidIsbn;
+
+import com.cgdp.library.exceptions.ValidationException;
+import java.time.LocalDate;
+import java.util.function.Supplier;
+
+public class Validator {
+
+    public static <T> T required(String paramName, T value) {
+        validate(() -> value == null, "%s must not be null", paramName);
+        return value;
+    }
+
+    public static String requiredNotNullOrBlank(String paramName, String value) {
```

Comment:
Let's just rename it to `requiredNotBlank`, required already implies that it should not be null
```suggestion
    public static String requiredNotBlank(String paramName, String value) {
```